README.md

```diff
@@ -66,6 +67,36 @@ mysite.local 127.0.0.1
 ```
 Now you can access the served content at `http://mysite.local:4568/`
 
+## Subscribing to S3 Event 
+
+You can subscribe to Put, Copy and Delete object events in the bucket.
+Please have a look at [Aws page](http://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html) for details of event object. 
+Apply filter function to subscribe to specific events.
+
+```
+var S3rver = require('s3rver');
```

Comment:
Please update this to `const`, we'd like to promote ES6 syntax were possible (even though a lot of our older code uses `var`)